### DISCUSSION TITLE
Github Education Benefits Approval Process Sucks

### DISCUSSION THREAD

**Discussion by anandypatil** [community/community#148247] [GitHub Education]
### Hi everyone,

I have applied for the Github developer pack more times than I can count at this point.

I fully qualify for the Github student developer pack. I'm sorry that my location doesn't show me on campus... it's Winter Break. I have never had trouble before verifying my status as a student on any other website; it is only Github that makes this process needlessly complicated. I have a verified email and that should be enough.

It asks for "proof why I am not on campus". I'm just sending in a screenshot of my calendar that says "Winter Break" because I genuinely have no idea what else I would submit.

I genuinely think Github does this on purpose to collect bad photos of frustrated students' faces holding up their student IDs to a camera.

Also, pro tip, if [your guide](https://github.com/orgs/community/discussions/111352#not-near-any-campus-location-for-selected-school) to do something fairly simple (and normally takes one click of a button/SSO log-in on any other website) is **this long**, then that speaks volumes at how bad your system is working for its users.

I am shocked at how bad this process is. Even worse, you can't even dispute the issue anymore with Github support. The approval website says to reach out to support, but if you select the ticket category that says `Application not approved (student)`, it literally just doesn't let you submit a ticket.

**Comment by samus-aran** [comment]
Hey @anandypatil I really appreciate you taking the time to jump on here and share your feedback and yes there is many improvements we can do to lower the barrier for genuine students making it easier to apply - we'll continue doing that. That being said... I do want to bring to your attention that with each rejection you receive reasons to help rectify your application and in this case with each submission, except your last successful one, we asked for 2FA to be enabled. 👀  This was the most likely biggest factor that affected your submissions.

> I'm sorry that my location doesn't show me on campus... it's Winter Break.

We're cool with folks being on vacation, we accommodate for this. That wasn't the main factor that your application(s) were rejected 🙂 

> I genuinely think Github does this on purpose to collect bad photos of frustrated students' faces holding up their student IDs to a camera.

😅 I guess making them into a 2025 calendar is out of the question? But, no, we don't like seeing authentic students and teachers feel deflated and we also don't like to see bad actors take advantage of the free offering our GitHub Education SDP partners provide.

> I have never had trouble before verifying my status as a student on any other website; it is only Github that makes this process needlessly complicated.

To help us, can you share what these websites are that provide, in your opinion, better academic verification? Perhaps we can reach out and gain a better understanding and insight?

Now, my biggest take away from your feedback is essentially elevating the need for 2FA to be enabled before a user submits their application, to which I will take back to the internal team to review. 

Even with the good-natured banter, we’re happy to see that you’ve been successfully verified! 🎉

**Comment by anandypatil** [comment]
I am happy that I finally got verified!

I will note that I did add two security keys to my account first thing I created it. However, apparently on Github, adding two security keys doesn’t automatically turn on 2FA? This confused me quite a bit and I didn’t realize until later that my 2FA wasn’t on; I kept logging out and back in and double checking to make sure I had my security keys added and it all looked good to me. I’ll also say that I did finally have it on before the last submission, and while I did log out and log back in, it still didn’t register I had turned it on and later rejected me (this was my second to last denial I believe).

Also, I will find those websites for you.

**Reply by samus-aran** [reply]
Thanks for sharing your 2FA experience, I'll include that to the feedback you've already raised. When it's enabled it shouldn't have any lag when submitting your SDP application 🤔 

Looking forward to checking those websites out. 👍
